modules/installation-custom-aws-vpc.adoc

```diff
@@ -54,6 +63,11 @@ ifdef::public[]
 * Create a public and private subnet for each availability zone that your cluster uses. Each availability zone can contain no more than one public and one private subnet. For an example of this type of configuration, see link:https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Scenario2.html[VPC with public and private subnets (NAT)] in the AWS documentation.
 +
 Record each subnet ID. Completing the installation requires that you enter these values in the `platform` section of the `install-config.yaml` file. See link:https://docs.aws.amazon.com/managedservices/latest/userguide/find-subnet.html[Finding a subnet ID] in the AWS documentation.
+ifdef::aws-outposts[]
+* In the availability zone, in which the AWS Outposts is installed, you must create a public and private subnets inside the Outpost, in addition to the ones mentioned above. This is achieved by indicating the Outposts ARN during the subnet creation.
```

Comment:
First comma should be removed, so the beginning is as follows:
In the availability zone in which...